Description: Fix crashes due to missing comma in call to gtk_tree_store_set()
Author: Frédéric Brière <fbriere@fbriere.net>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=764510
Forwarded: https://github.com/Gnucash/gnucash/pull/17
Reviewed-by: Sébastien Villemot <sebastien@debian.org>
Last-Update: 2014-10-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/gnome/dialog-sx-since-last-run.c
+++ b/src/gnome/dialog-sx-since-last-run.c
@@ -537,8 +537,8 @@ gsslrtma_populate_tree_store(GncSxSlrTre
                                            SLR_MODEL_COL_VARAIBLE_VALUE, tmp_str->str,
                                            SLR_MODEL_COL_INSTANCE_VISIBILITY, FALSE,
                                            SLR_MODEL_COL_VARIABLE_VISIBILITY, TRUE,
-                                           SLR_MODEL_COL_INSTANCE_STATE_SENSITIVITY, FALSE
-                                           - 1);
+                                           SLR_MODEL_COL_INSTANCE_STATE_SENSITIVITY, FALSE,
+                                           -1);
                         g_string_free(tmp_str, TRUE);
                     }
                     g_list_free(vars);
